src/plugins/geomap/_base.scss

```diff
@@ -279,7 +279,7 @@
 }
 
 .ol-viewport .ol-unselectable {
-	-khtml-user-select: none;
+	/* -khtml-user-select: none; sasslint doesn't like this */
```

Comment:
@jvanulde I presume this change was made to avoid https://travis-ci.org/wet-boew/wet-boew/builds/296480431#L2266?

I'd recommend removing that line rather than commenting it out. There isn't any value in keeping it in a commented-out state in the long run.

Btw I believe WET currently only officially supports vendor prefixes for Webkit/Blink (``-webkit-``), Firefox (``-moz``) and IE/Edge (``-ms-``). AFAIK Konqueror (``-khtml-``) and classic Opera (``-o-``) prefixes aren't supported. Maybe @LaurentGoderre or @nschonni could definitively confirm.